src/Navigation/Xamarin.Forms/Navigation.Xamarin.Forms/NavigationServiceExtensions.cs

```diff
@@ -227,5 +181,20 @@ public static class NavigationServiceExtensions
 
             return registrationInfo;
         }
+
+
+
+        private static void ThrowIfNavigationServiceIsOfWrongType(
+            INavigationService service,
+            out NavigationService navigationService)
+        {
+            if (!(service is NavigationService wpfService))
+            {
+                throw new InvalidOperationException(
+                    $"This extension method can only be used with {nameof(NavigationService)} of type {typeof(NavigationService).FullName}!");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Split this 142 characters long line (which is greater than 120 authorized).](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5598919)